ci/buildspec-extension.yml

```diff
@@ -0,0 +1,31 @@
+version: 0.2
+
+env:
+  variables:
+    SKLEARN_FRAMEWORK_VERSION: "0.23-1"
+    EXTENSION_FRAMEWORK_VERSION: "0.2-1"
+
+phases:
+  install:
+    runtime-versions:
+      docker: 18
+      python: 3.7
+  pre_build:
+    commands:
+    - echo Logging in to Amazon ECR...
+    - $(aws ecr get-login --no-include-email --region $AWS_DEFAULT_REGION)
+  build:
+    commands:
+    - echo Build started on `date`
+    - echo Building the Docker image...
+    - docker build -t sklearn-base:$FRAMEWORK_VERSION-cpu-py3  -f docker/$SKLEARN_FRAMEWORK_VERSION/base/Dockerfile.cpu .
+    - pip install wheel setuptools
+    - python setup.py bdist_wheel
+    - docker build -t preprod-sklearn:$FRAMEWORK_VERSION-cpu-py3 -f docker/$SKLEARN_FRAMEWORK_VERSION/final/Dockerfile.cpu .
```

Comment:
`preprod-sklearn:$FRAMEWORK_VERSION-cpu-py3` should be `preprod-sklearn:$SKLEARN_FRAMEWORK_VERSION-cpu-py3`